docs/contributors/code/release.md

```diff
@@ -34,6 +34,10 @@ To release a release candidate (RC) version of the plugin, enter `rc`. To releas
 
 This will trigger a GitHub Actions (GHA) workflow that bumps the plugin version, builds the Gutenberg plugin .zip file, creates a release draft, and attaches the plugin .zip file to it. This part of the process typically takes a little under six minutes. You'll see that workflow appear at the top of the list, right under the blue banner. Once it's finished, it'll change its status icon from a yellow dot to a green checkmark. You can follow along in a more detailed view by clicking on the workflow.
 
+#### Publishing the @wordpress packages to NPM
+
+As part of the release candidate (RC) process, all of the `@wordpress` packages are published to NPM. You may see messaging after the ["Build Gutenberg Plugin Zip" action](https://github.com/WordPress/gutenberg/actions/workflows/build-plugin-zip.yml) action has created the draft release that the ["Publish npm packages"](https://github.com/WordPress/gutenberg/actions/workflows/publish-npm-packages.yml) action requires someone with appropriate permissions to trigger the action. This is not the case as this process is automated and it will automatically run after the release notes are published.
```

Comment:
> This is not the case as this process is automated and it will automatically run after the release notes are published.

@ryanwelcher I found it hard to understand this last part. Is there any further clarification that can be added?

Maybe something like "As the release coordinator you don't need to take any action"?(this is my interpretation of what the sentence means, but I wasn't sure).

I'm looking to update the release docs at the moment, so let me know and I can include a small update to this part.

Reply:
@talldan you are correct. Feel free to make any clarifications you think are appropriate Thanks!

Reply:
Thanks, [I made some changes here](https://github.com/WordPress/gutenberg/pull/47480)